lib/improver/utilities/cube_manipulation.py

```diff
@@ -247,6 +250,9 @@ def equalise_cubes(cubes_in, merging=True):
         merging (boolean):
             Flag for whether the equalising is for merging
             as slightly different processing is required.
+        model_id_attr (str):
+            Identification string of the model configuration.
+
```

Comment:
Need a docstring entry for the merging argument please. I'm not quite sure I follow the logic of why this argument is being applied. Could you update the docstring or comments to explain?

Reply:
This isn't anything I've changed or added, but will look into it....There is a docstring for the merging argument, but I take it you mean that it requires improvement?

Reply:
Sorry, my bad, I completely missed that!  In which case could you just reorder the docstring so that it matches the argument order please (so that silly people like me can see what's right in front of them...)

Reply:
Changed the order of the docstring.